src/SOS/Strike/sos_unixexports.src

```diff
@@ -12,6 +12,7 @@ DumpClass
 DumpDelegate
 DumpDomain
 DumpGCData
+DumpHttpRequests
```

Comment:
Sorry, I was wrong about needing this export for Linux and MacOS under lldb. It is the reason for the MacOS build failures in CI.  For managed only commands like this one the export isn't needed just the soscommand.cpp changes.